### DISCUSSION TITLE
Property "extend" missing in V2

### DISCUSSION THREAD

**Discussion by zanzara** [quasarframework/quasar-ui-qmarkdown#376] [Ideas]
i would need  this prop in qmarkdown-v2
Why was it discarded?  
Could it be implemented somehow again in v2 ? 

PS.: I cannot follow exactly how it was done in v2.  Otherwise I would make a PR.

Do you have at least some hints for that?
Thx!

**Comment by nucle** [comment]
Hi,
please use the plugin property.
Example can be found here: 
https://github.com/quasarframework/quasar-ui-qmarkdown/issues/374

BR,
nucle
